.github/workflows/coverage.yml

```
@@ -39,10 +39,10 @@ jobs:
                 libfftw3-dev libhdf5-dev libblas-dev liblapack-dev
 
       - name: Build
-        run: /bin/bash mfc.sh build -j $(nproc) --gcov
+        run: /bin/bash mfc.sh build -v -j $(nproc) --gcov
```

actionlint (1.7.10)

[error] shellcheck reported issue in this script: SC2046:warning:1:30: Quote this to prevent word splitting

(shellcheck)